### PR TITLE
Made threadPoolSize a member variable.

### PR DESCRIPTION
**Pull request by boneill42**
Made threadPoolSize a member variable.

### CODE REVIEW

File: src/main/java/com/hmsonline/cassandra/triggers/TriggerTask.java [hunk 1]
```diff
@@ -9,13 +9,14 @@
 import org.slf4j.LoggerFactory;
 
 public class TriggerTask implements Runnable {
+    private static Logger logger = LoggerFactory.getLogger(TriggerTask.class);
```
Comment (ptgoetz) [inline src/main/java/com/hmsonline/cassandra/triggers/TriggerTask.java:12]: Very minor... but I'd make this static final and name it LOG... (PMD/CheckStyle would likely bark about it).

**Comment by ptgoetz** [comment]
Works for me.

File: src/main/java/com/hmsonline/cassandra/triggers/TriggerTask.java (hunk 1 above)
Comment (boneill42) [inline src/main/java/com/hmsonline/cassandra/triggers/TriggerTask.java:12]: Good point.  Not sure where it started, but we're doing this everywhere.

Isaac/Shantha, what do you think?

-brian

---

Brian O'Neill
Lead Architect, Software Development
Health Market Science | 2700 Horizon Drive | King of Prussia, PA 19406
p: 215.588.6024blog: http://weblogs.java.net/blog/boneill42/
blog: http://brianoneill.blogspot.com/

On 3/27/12 9:17 PM, "P. Taylor Goetz"
<reply+p-612838-39eb2c923c8294398488b5a9d783e468467ca293-897229@reply.githu
b.com> wrote:

Comment (sandrews) [inline src/main/java/com/hmsonline/cassandra/triggers/TriggerTask.java:12]: I agree, infact we need to .getCanonicalName() on the class too

**Pull request merged**